datafusion/expr/src/type_coercion/aggregates.rs

```diff
@@ -621,17 +592,6 @@ mod tests {
                 assert_eq!(*input_type, result.unwrap());
             }
         }
-        // test sum
```

Comment:
I think we don't need coercion test, because they are handled in slt query already.